src/main/groovy/life/qbic/datamodel/accounting/CostItem.groovy

```diff
@@ -0,0 +1,43 @@
+package life.qbic.datamodel.accounting
+
+/**
+ * An item with associated costs and quantity
+ *
+ * Cost items describe services and their price. They are the building blocks of invoices and offers.
+ * The unit price is always provided in euros.
+ * @since: 1.9.0
+ * @author: Tobias Koch
+ *
+ * * */
+class CostItem {
```

Comment:
NIT: maybe change to PackageItem to keep the vocabulary of the process?

Reply:
Package item implies that the item is inside a package but it is actually the other way around as far as I understood it. For now, I only created a CostItem to keep it as simple as possible. We could differentiate further in the future if that is desired.